### PR TITLE
Fix #237 "g_get_current_time() is deprecated"

### LINKED ISSUE #237: g_get_current_time() is deprecated
g_get_current_time() was deprected in Glib 2.62.
The documentation suggest using g_get_get_real_time() instead, which has been available since glib 2.28. Ubuntu 16.04 has Glib 2.48.

### PR DESCRIPTION
**Pull request by yashi**
Hi,

This PR is to fix #237.  In the course of fixing it I found a few low hanging fruits, which are included in this PR as well.

Hope you like it.

### CODE REVIEW

File: src/tle-update.c [hunk 1]
```diff
@@ -396,10 +396,10 @@ void tle_update_from_files(const gchar * dir, const gchar * filter,
             /* store time of update if we have updated something */
             if ((updated > 0) || (newsats > 0))
             {
-                GTimeVal        tval;
+		gint64          now;
```
Comment (csete) [inline src/tle-update.c:399]: In the future, please keep 4-spaces indentation.

File: src/tle-update.c [hunk 2]
```diff
@@ -396,10 +396,10 @@ void tle_update_from_files(const gchar * dir, const gchar * filter,
...
 
-                g_get_current_time(&tval);
-                sat_cfg_set_int(SAT_CFG_INT_TLE_LAST_UPDATE, tval.tv_sec);
+		now = g_get_real_time() / G_USEC_PER_SEC;
```
Comment (csete) [inline src/tle-update.c:401]: Same as above.

**Review by csete — APPROVED** [review]

**Review by csete — APPROVED** [review]

**Review by csete — APPROVED** [review]

**Review by csete — APPROVED** [review]

**Review by csete — APPROVED** [review]

**Review by csete — APPROVED** [review]

**Comment by csete** [comment]
Thanks for the fixes @yashi. 
In the future, I would prefer to have different fixes in different pull request otherwise it can become difficult to review and update them separately.

File: src/tle-update.c (hunk 1 above)
Comment (yashi) [reply src/tle-update.c:399]: ah, do you want all spaces? It looks like it. I'll make sure I'll follow it.  Would you take a commit to fix the whitespace errors?

While we are at it, what do you think about trailing whitespaces.  There are so many.  Do you think we can remove them in one go or gradually removing while fixing the line with it?

> Furthermore, until we get a formal coding style guide, please try to follow the coding
style used in the surrounding code.

What do you think about providing a config file for clang-format or uncrustify?

Is it ok to talk about it here, or create an issue about it? How about enabling github discussion?

Comment (csete) [reply src/tle-update.c:399]: > While we are at it, what do you think about trailing whitespaces. There are so many. Do you think we can remove them in one go or gradually removing while fixing the line with it?

I prefer that changes that fix the formatting or style of previously written code to be done separately.

> What do you think about providing a config file for clang-format or uncrustify?

There is a config file for the [indent](https://www.gnu.org/software/indent/manual/indent.html) tool, however, I don't remember whether all files have been indented already. I might switch to clang-format or some other tool in the future, but all these kind of things are time-consuming and I have other priorities for the time being.

> Is it ok to talk about it here, or create an issue about it? How about enabling github discussion?

Thanks for the tip, I wasn't aware of github discussions but I will check it out.

**Pull request merged**